src/app-layer.h

```diff
@@ -16,6 +16,10 @@
  */
 
 /**
+ * \file
+ *
+ * Application layer handling and protocols implementation
+ *
  * \author Victor Julien <victor@inliniac.net>
  * \author Anoop Saldanha <anoopsaldanha@gmail.com>
```

Comment:
One last comment, I'd say, so we keep documentation like what we see in other files (check https://github.com/OISF/suricata/blob/master/src/app-layer-ftp.c#L18-L26, for instance):
```suggestion
 * \file
 *
 * \author Victor Julien <victor@inliniac.net>
 * \author Anoop Saldanha <anoopsaldanha@gmail.com>
 *
 * Application layer handling and protocols implementation
```

Could you ensure that all documentation you've added is following this format? One reason for this request is that if we can keep such things as standardized as possible, it's easier for newcomers to know how to do things, as if they check the code for examples, the pattern will be clear for them :)

It also looks better, in general, if all code is following the same style - as much as possible.